dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -467,10 +467,16 @@ private void resolve(Object installObject) {
         Collection<List<RepositoryResource>> resolveResult = null;
         RepositoryResolver resolver = null;
         Collection<String> featuresResolved = new ArrayList<String>();
-
+        String edition = null;
         try {
             for (ProductInfo productInfo : ProductInfo.getAllProductInfo().values()) {
                 productDefinitions.add(new ProductInfoProductDefinition(productInfo));
+                edition = productInfo.getEdition();
```

Comment:
There could be multiple ProductInfo objects, where one replaces the other (see ProductInfo.getReplaces() and ProductInfo.getReplacedBy()).  If "replacedBy" is null, then that is the actual product.

So we can do something like
```
if (productInfo.getReplacedBy() == null && productInfo.getId().equals("io.openliberty") {
    isOpenLiberty = true;
}
```